src/kili/services/export/format/coco/__init__.py

```diff
@@ -227,7 +227,7 @@ def _get_coco_images_and_annotations(
         coco_image = _CocoImage(
             id=asset_i,
             license=0,
-            file_name=str("data" / Path(file_name.parts[-1])),
+            file_name=str("data" + "/" + file_name.parts[-1]),
```

Comment:
```suggestion
            file_name=str(PosixPath("data") / file_name.parts[-1]),
```
should be working

Reply:
indeed this is `PurePosixPath` . I let you choose the solution. In any case, you should add a comment to says that the posix format is imposed by the COCO format.